src/mcts/search.cc

```diff
@@ -595,8 +595,10 @@ EdgeAndNode Search::GetBestChildWithTemperature(Node* parent,
             root_limit.end()) {
       continue;
     }
-    if (edge.GetN() + offset > max_n) max_n = edge.GetN() + offset;
-    if (edge.GetQ(fpu) > max_eval) max_eval = edge.GetQ(fpu);
+    if (edge.GetN() + offset > max_n) {
+      max_n = edge.GetN() + offset;
+      if (edge.GetQ(fpu) > max_eval) max_eval = edge.GetQ(fpu);
```

Comment:
We probably don't want the conditional assignment here? If the best/most-visited move's Q drops, it might not be able to replace the max_eval.

Reply:
Probably no need for condition indeed.

Reply:
Removed